### ISSUE TITLE
What is flickr25k_label_20015.npy file?

### ISSUE THREAD

**Issue by xueliang** [yolo2233/cross-modal-hasing-playground#2]
Dear Author,
    What is the flickr25k_label_20015.npy?  the label indexing of each data? Thanks.

**Comment by yolo2233** [comment]
Yes. It is a N x M numpy matrix. N denotes the number of instances. M denotes the number of classes. @xueliang
